security/medusa/l3/comm.c

```diff
@@ -4,12 +4,72 @@
 #include <linux/medusa/l3/arch.h>
 #include <linux/medusa/l3/registry.h>
 #include <linux/medusa/l3/server.h>
+#include <linux/medusa/l4/interface.h>
+
 #include "l3_internals.h"
 
+medusa_answer_t development_converter(authserver_answer_t code)
+{
+	medusa_answer_t retval;
+	switch(code) {
+		case AUTHS_ERR:
+			retval = MED_ERR;
+			break;
+		case AUTHS_ALLOW:
+		case AUTHS_DBG_ALLOW:
+		case AUTHS_FORCE_ALLOW:
+		case AUTHS_NOT_REACHED:
+			retval = MED_OK;
+			break;
+		case AUTHS_DENY:
+		case AUTHS_SKIP:
+			retval = MED_NO;
+			break;
+		default:
+			retval = MED_OK;
+	}
+	return retval;
+}
+
+medusa_answer_t production_converter(authserver_answer_t code)
+{
+	medusa_answer_t retval;
+	switch(code) {
+		case AUTHS_ERR:
+			retval = MED_ERR;
+			break;
+		case AUTHS_ALLOW:
+		case AUTHS_FORCE_ALLOW:
+			retval = MED_OK;
+			break;
+		case AUTHS_DENY:
+		case AUTHS_NOT_REACHED:
+		case AUTHS_SKIP:
```

Comment:
should be MED_SKIP

Reply:
As we discussed, MED_SKIP is now deprecated and all references will result in compile error.

Reply:
so we will resolve this in other way. we cannot make wrong conversion(s)!

Reply:
I agree we cannot delete it since it can come from auth server as valid answer. What about Medusa code for not supported answer? I think it can possibly happen, mainly when there will be people creating its own authorization servers. I think it would fit, since right now deprecated codes are not supported, such a conversion may not be confusing.

What do you think? We should decide what should be the value of this code and how should Medusa behave.